pkgs/development/python-modules/cherrypy-cors/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonRelaxDepsHook
+, pytestCheckHook
+, setuptools
+, httpagentparser
+, cherrypy
+}:
+buildPythonPackage rec {
+  pname = "cherrypy-cors";
+  version = "1.7.0";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-gzhM1mSnq4uat9SSb+lxOs/gvONmXuKBiaD6BLnyEtY=";
+  };
+
+  nativeBuildInputs = [
+    pythonRelaxDepsHook
+    pytestCheckHook
```

Comment:
`pytestCheckHook` is a test dependency, so that it should go to `nativeCheckInputs`.